extensions/ql-vscode/src/run-remote-query.ts

```diff
@@ -36,21 +35,23 @@ export default async function runRemoteQuery(credentials: Credentials, uri?: Uri
 
   const ref = config.ref || 'main';
   const language = config.language;
-  const repositories = JSON.stringify(config.repositories);
+  const repositories = config.repositories;
 
   try {
-    await octokit.rest.actions.createWorkflowDispatch({
-      owner: OWNER,
-      repo: REPO,
-      workflow_id: WORKFLOW_ID,
-      ref: ref,
-      inputs: {
-        language,
-        repositories,
-        query,
-        token
+    await octokit.request(
+      'POST /repos/:owner/:repo/code-scanning/codeql/queries',
+      {
+        owner: OWNER,
+        repo: REPO,
```

Comment:
Presumably when this is released, we will no longer be pointing to internal repositories?

It's not great that we have some code in this repo that is only usable with proper permissioning, but since this is guarded by a canary it's fine.